docs/README.md

```diff
@@ -39,3 +39,49 @@ $ GIT_USER=<Your GitHub username> yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+
+### How to add tabs for Programming Languages
+
+Att the top of the `.md` file, ensure having the following: 
+
+```js
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+Use the following snippet for tabs:
+
+<pre>
+<xmp lang=html>
+<Tabs groupId="prog-lang" queryString>
+
+  <TabItem value="javascript" label="JavaScript" default 
+  	attributes={{className: "javascript-tab"}}>
+
+```javascript
+const { Web3 } = require('web3');
+// ...
+```
+
+  </TabItem>
+  
+  <TabItem value="typescript" label="TypeScript"
+  	attributes={{className: "typescript-tab"}}>
+
+```typescript
+import { Web3 } from 'web3';
+// ...
+```
+
+  </TabItem>
+</Tabs>
+</xmp>
+</pre>
+
+Notes: 
+- This will put the JavaScript tab as the default. This is chosen because a JavaScript project needs less configurations. However, if the tutorial steps involves installing and configuring TypeScript, put the `default` attribute on the TypeScript tab instead.
```

Comment:
```suggestion
- This will put the JavaScript tab as the default. This is chosen because a JavaScript project needs less configurations. However, if the tutorial steps involve installing and configuring TypeScript, put the `default` attribute on the TypeScript tab instead.
```